linera-views/src/journaling.rs

```diff
@@ -52,21 +62,20 @@ fn get_journaling_key(base_key: &[u8], tag: u8, pos: u32) -> Result<Vec<u8>, bcs
 /// Low-level, asynchronous direct write key-value operations with simplified batch
 #[async_trait]
 pub trait DirectWritableKeyValueStore<E> {
-    /// The maximal number of items in a simplified transaction
-    const MAX_TRANSACT_WRITE_ITEM_SIZE: usize;
+    /// The maximal number of items in a batch.
+    const MAX_BATCH_SIZE: usize;
 
-    /// The maximal number of bytes of a simplified transaction
-    const MAX_TRANSACT_WRITE_ITEM_TOTAL_SIZE: usize;
+    /// The maximal number of bytes of a batch.
+    const MAX_BATCH_TOTAL_SIZE: usize;
 
     /// The maximal size of values that can be stored.
     const MAX_VALUE_SIZE: usize;
 
     /// The batch type.
-    type SimpBatch: SimplifiedBatch + Serialize + DeserializeOwned + Default;
+    type Batch: SimplifiedBatch + Serialize + DeserializeOwned + Default;
```

Comment:
Maybe the notion of `SimplifiedBatch` is badly named. Possibly a better name is `NonCollidingBatch`.

Reply:
Let me think about a new name needed. The trait `SimplifiedBatch` is more about batch processing and starting to look rather independent from the concept of (simple) unordered batch.

Reply:
And yes maybe the name "unordered" could be improved. It took me a while to remember that it was about disjoint (non-conflicting) keys.